src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1153,6 +1180,9 @@ internal StyleViewModel(OptionSet optionSet, IServiceProvider serviceProvider) :
 
             // Field preferences.
             CodeStyleItems.Add(new BooleanCodeStyleOptionViewModel(CodeStyleOptions.PreferReadonly, ServicesVSResources.Prefer_readonly, s_preferReadonly, s_preferReadonly, this, optionSet, fieldGroupTitle));
+
+            // Member preferences (field/method/property/event).
+            CodeStyleItems.Add(new BooleanCodeStyleOptionViewModel(CodeStyleOptions.AvoidUnusedMembers, ServicesVSResources.Avoid_unused_members, s_avoidUnusedMembers, s_avoidUnusedMembers, this, optionSet, memberGroupTitle));
```

Comment:
if we do this, we shoudl probably make 'MakeFieldReadonly' go here.  As well as the stuff about requiring modifiers.